compile error, missing parameter -lnvidia-ml

i think theres a problem with your Makefile.

heres the error im getting:

g++ -I/usr/local/cuda//include -L/usr/local/cuda//lib64/ cuda-smi.cpp -lnvidia-ml -lcudart_static -lpthread -ldl -lrt -o cuda-smi
/usr/bin/ld: cannot find -lnvidia-ml
collect2: error: ld returned 1 exit status
make: **\* [cuda-smi] Fehler 1

some sys specs:

cuda toolkit version 7.0 from nvidia.com resides in /usr/local/cuda
cuda management library newest installed in global scope (as of 30-5-2015)
g++ version (Ubuntu 4.8.2-19ubuntu1) 4.8.2
sys Description: Ubuntu 14.04.2 LTS
nvidia module version: nvidia-349, 349.16, 4.0.1, x86_64: installed

hope youre able to help me out.
greetings
Oliver

--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,8 @@
 CUDA_HOME=/usr/local/cuda/
 
-CXXFLAGS+=-I$(CUDA_HOME)/include -L$(CUDA_HOME)/lib64/
+NVIDIA_DRIVER_VERSION=$(shell cat /proc/driver/nvidia/version | sed -e 2d | sed -E 's,.* ([0-9]*)\.([0-9]*) .*,\1,')
+
+CXXFLAGS+=-I$(CUDA_HOME)/include -L$(CUDA_HOME)/lib64/ -L/usr/lib/nvidia-$(NVIDIA_DRIVER_VERSION)/
 
 cuda-smi: cuda-smi.cpp nvml.h
 	$(CXX) $(CXXFLAGS) $< -lnvidia-ml -lcudart_static -lpthread -ldl -lrt -o $@

